website/src/react-native/showcase.js

```diff
@@ -868,7 +868,14 @@ var apps = [
     icon: 'https://s3-ap-northeast-1.amazonaws.com/sw-misc/sharewis3_app.png',
     link: 'https://itunes.apple.com/jp/app/id585517208',
     author: 'ShareWis Inc.'
-  }
+  },
+	{
+		"name": "婚格",
+		"desc": "婚格-让结婚更简单",
+		"ios": "https://itunes.apple.com/cn/app/hun-ge-rang-jie-hun-geng-jian/id1062457289?mt=8",
+		"android": "http://7xsmao.com2.z0.glb.qiniucdn.com/app-release.apk",
```

Comment:
There's no such property as `ios/android`. Please check this out for reference https://github.com/mozillo/react-native/blob/patch-1/website/src/react-native/showcase.js#L24-L25